libraries/tools/kotlin-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/targets/native/spm/KotlinSwiftPackagePlugin.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+@file:Suppress("PackageDirectoryMismatch") // Old package for compatibility
```

Comment:
Remove this annotation and move sources to `org.jetbrains.kotlin.gradle.plugin.spm` package